benchmarks/cluster/prometheus-operator-values.yml

```diff
@@ -0,0 +1,49 @@
+alertmanager:
+  enabled: false
+
+grafana:
+  adminPassword: camunda
```

Comment:
Thats why I havent added it the last time to the benchmark folder :sweat_smile:

Reply:
Yeah I thought about it, but it's not like it's a good password anyway. You'd still need to know the private IP of our Grafana instance. We could also always make it private again and you'd need to port-forward it, meaning you'd need access to the cluster, so really even having a password is kind of pointless :sweat_smile: